app/views/file_upload_errors.scala.html

```diff
@@ -14,37 +14,43 @@
  * limitations under the License.
  *@
 
-@import utils.ERSUtil
-@import config.ApplicationConfig
-
-@this(config: ApplicationConfig)
+@import _root_.utils.ERSUtil
+@import _root_.config.ApplicationConfig
+@import views.html.govuk_wrapper
+@import views.html.templates.reference
+
+@this(
+		govuk_wrapper: govuk_wrapper,
+		govukBackLink: GovukBackLink,
+		govukInsetText: GovukInsetText,
+		hmrcPageHeading: HmrcPageHeading
+)
 
 @(requestObject: RequestObject, fileType: String)(implicit request: Request[AnyRef], messages: Messages, ersUtil: ERSUtil, appConfig: ApplicationConfig)
-@import views.html.templates._
 
-@ers_main(title = messages("file_upload_errors.title") + " " + messages("ers.title.postfix")) {
+@govuk_wrapper(title = messages("file_upload_errors.title") + " " + messages("ers.title.postfix")) {
 
 	@reference(requestObject.getPageTitle)
 
-	@if(fileType == ersUtil.OPTION_ODS){
-		<p class="font-xsmall"><a href="@routes.FileUploadController.uploadFilePage" class="link-back">@Html(messages("ers.back"))</a></p>
-	} else {
-		<p class="font-xsmall"><a href="@routes.CsvFileUploadController.uploadFilePage" class="link-back">@Html(messages("ers.back"))</a></p>
-	}
-
-	@header(messages("file_upload_errors.title"))
-
-	<div class="panel-indent">
-		@if(fileType == ersUtil.OPTION_ODS) {
-			<p>@Html(messages("file_upload_errors.para1")) <a href="@routes.FileUploadController.uploadFilePage()">@Html(messages("file_upload_errors.upload_link"))</a>.</p>
-		} else {
-			<p>@Html(messages("file_upload_errors.para1")) <a href="@routes.CheckCsvFilesController.checkCsvFilesPage()">@Html(messages("file_upload_errors.upload_link"))</a>.</p>
-		}
-	</div>
-
-	<h2>@Html(messages("file_upload_errors.go_fix"))</h2>
-
-	<p><a href="/check-your-ers-files">@Html(messages("file_upload_errors.check_file"))</a> @Html(messages("file_upload_errors.info"))</p>
-
-
+	@hmrcPageHeading(PageHeading(
+		text = messages("file_upload_errors.title")
+	))
+
+	@govukInsetText(InsetText(
+		content = HtmlContent(
```

Comment:
just to avoid doubling this up, I think it could be done like:
```
content = {
   val link: String = if (fileType == ersUtil.OPTION_ODS) routes.FileUploadController.uploadFilePage.url else routes.CheckCsvFilesController.checkCsvFilesPage.url
   messages("stringStuff") <a href=link> messages ("otherStringStuff")
}

```

Reply:
Nice, good call. Changing this.